index.html

```diff
@@ -70,100 +74,103 @@ <h1 class="headline-title">
     </section>
 
     <!-- work section -->
-    <section class="work-container">
+    <section class="work-container container">
       <div class="work-heading">
         <h2>My Recent Works</h2>
         <div class="line"></div>
       </div>
-      <div class="grid-container">
-        <div class="grid-card">
-          <div class="card-img"></div>
-          <div class="box">
-            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
-            <ul class="tags">
-              <li class="tag-item">Ruby on rails</li>
-              <li class="tag-item">CSS</li>
-              <li class="tag-item">JavaScript</li>
-              <li class="tag-item">HTML</li>
-            </ul>
-            <button type="submit">See Project</button>
+      <div class="grid-container work-sub-container">
```

Comment:
- [ ] [REQUIRED] The previous reviewer requested that you use CSS Grid on the web page and that was a bit not specific. I notice you implemented it in the navbar. Actually, that should be implemented here in this section as it is a core requirement for this project. Kindly use the grid to improve the responsiveness of the project container.